project/cmake/addons/bootstrap/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+project(kodi-addons-bootstrap)
+
+cmake_minimum_required(VERSION 2.8)
+
+list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR})
+
+# make sure CMAKE_INSTALL_PREFIX is properly set
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT OR NOT CMAKE_INSTALL_PREFIX)
+  set(CMAKE_INSTALL_PREFIX "${PROJECT_SOURCE_DIR}/../addons")
+endif()
+list(APPEND CMAKE_PREFIX_PATH ${CMAKE_INSTALL_PREFIX})
+
+# figure out where the build directory is located
+if(NOT BUILD_DIR)
+  set(BUILD_DIR "${CMAKE_BINARY_DIR}/build")
+else()
+  file(TO_CMAKE_PATH "${BUILD_DIR}" BUILD_DIR)
+endif()
+get_filename_component(BUILD_DIR "${BUILD_DIR}" ABSOLUTE)
+
+# figure out which addons to bootstrap (defaults to all)
+if(NOT ADDONS_TO_BUILD)
+  set(ADDONS_TO_BUILD "all")
+  message(STATUS "Bootstrapping all addons")
+else()
+  message(STATUS "Bootstrapping following addons: ${ADDONS_TO_BUILD}")
+endif()
+
+# make sure that the addon type has been specified
+if(NOT ADDON_TYPE)
+  message(FATAL_ERROR "No addon type specified")
+endif()
+
+# make sure that the addon definitions repository has been specified
+if(NOT ADDON_DEFINITIONS_REPOSITORY)
+  message(FATAL_ERROR "No addons definition repository specified")
```

Comment:
Whats the advantage of having multiple repos? 
I'd much prefer a single meta repo that holds all addons, if possible separated into addon classes/types. Imho this is easier to handle and to not lose overview.